information-schema/information-schema-resource-groups.md

```diff
@@ -64,11 +64,11 @@ SELECT * FROM information_schema.resource_groups WHERE NAME = 'rg1'; -- 查看
 ```
 
 ```sql
-+------+------------+----------+-----------+
-| NAME | RU_PER_SEC | PRIORITY | BURSTABLE |
-+------+------------+----------+-----------+
-| rg1  |       1000 | MEDIUM   | NO        |
-+------+------------+----------+-----------+
++------+------------+----------+-----------+-------------+
+| NAME | RU_PER_SEC | PRIORITY | BURSTABLE | QUERY_LIMIT |
++------+------------+----------+-----------+-------------+
+| rg1  |      1000  | MEDIUM   | NO        | NULL        |
```

Comment:
NIT: Why the align of `RU_PER_SEC` is different from other fields? Looks a bit weird.

Reply:
fixed